.flowconfig

```diff
@@ -0,0 +1,13 @@
+[ignore]
+.*/tests/.*
+.*node_modules/.*
+
+[include]
+src/
+
+[libs]
+interfaces/
```

Comment:
We don't have this dir yet, but we can populate it with annotations for 3rd party lib stuff when we get to files that use them.